### ISSUE TITLE
Mapnik tiles are blank

### ISSUE THREAD

**Issue by jgneff** [bparmentier/OpenBikeSharing#51]
A few weeks ago, the Mapnik background tiles started showing up blank. I preferred Mapnik over the other two map layer choices because they also have problems. The OpenCycleMap view is stamped with *API Key Required*, and the OSMPublicTransport map shows only the public transportation lines without any other features.

See below for current examples of all three map layers in downtown Vancouver, British Columbia, Canada.

| Mapnik | OpenCycleMap | OSMPublicTransport |
|:------:|:------------:|:------------------:|
| ![Mapnik view](https://user-images.githubusercontent.com/1413266/65276208-19155700-dadc-11e9-8675-e796e67a7005.png) | ![OpenCycleMap view](https://user-images.githubusercontent.com/1413266/65276213-1dda0b00-dadc-11e9-8c7d-0b5cab5a6cce.png) | ![OSMPublicTransport view](https://user-images.githubusercontent.com/1413266/65276219-216d9200-dadc-11e9-998d-ba7b1c9b11e2.png) |

**Comment by fabmazz** [comment]
Which handset and version of android do you use?

**Comment by jgneff** [comment]
I have two phones with the problems shown in the screenshots:

* [Motorola Moto G 2014](https://www.gsmarena.com/motorola_moto_g_(2nd_gen)-6647.php) running /e/OS 0.7, which is a fork of LineageOS based on Android 7.1.2.
* [Motorola Moto G7 Play](https://www.gsmarena.com/motorola_moto_g7_play-9532.php) running the official Android 9 from Google and Motorola.

**Comment by fabmazz** [comment]
It's possible it's a permission problem, can you check that you have
enabled the storage permission?

Il Sab 5 Ott 2019, 18:26 John Neffenger <notifications@github.com> ha
scritto:

**Comment by jgneff** [comment]
Yes, Location and Storage permissions are enabled for the app on both phones:

![permissions](https://user-images.githubusercontent.com/1413266/66257954-50c70480-e754-11e9-99a9-f936fe312d60.png)

**Comment by francoisfds** [comment]
Hi ! I recently noticed this app and found it quite useful, congrats to @bparmentier :+1:

Unfortunately my device is also concerned by this Mapnik issue. It turns out that osmdroid (the library underneath the map rendering) addressed this issue in recent releases, so I've suggested a pull request to get OpenBikeSharing up to date. I would be grateful if you could take a look at it.

Thank you again for bringing this app to the community !

**Comment by francoisfds** [comment]
Hello @bparmentier.

I assume you didn't have time to get into the pull request #52 I submitted. Be sure that I tested it, I haven't seen regressions and I am here to explain if you have any technical question on the changes.
If you still are nearby, please consider to have a look at all the opened PR, and even tag a version so others can enjoy the app again.

Take care,
François.
